### PR TITLE
Fix (internal) documentation for pytester's LineComp

### PR DESCRIPTION
**Pull request by blueyed**

### CODE REVIEW

File: src/_pytest/pytester.py [hunk 1]
```diff
@@ -1283,21 +1283,21 @@ def spawn(self, cmd: str, expect_timeout: float = 10.0) -> "pexpect.spawn":
...
 
+        Lines are matched using :func:`LineMatcher.fnmatch_lines`.
         """
         __tracebackhide__ = True
         val = self.stringio.getvalue()
         self.stringio.truncate(0)
         self.stringio.seek(0)
         lines1 = val.split("\n")
-        return LineMatcher(lines1).fnmatch_lines(lines2)
+        LineMatcher(lines1).fnmatch_lines(lines2)
```
Comment (RonnyPfannschmidt) [inline src/_pytest/pytester.py]: why remove the return?

Comment (blueyed) [reply src/_pytest/pytester.py]: hint: `LineMatcher.fnmatch_lines` is type-hinted (it returns None).

Comment (nicoddemus) [reply src/_pytest/pytester.py]: > hint: ...

I don't think a "hint" is necessary, just explaining the reason is enough.

Comment (blueyed) [reply src/_pytest/pytester.py]: Ok:

> `fnmatch_lines` returns nothing (see 4 clicks below)

Comment (nicoddemus) [reply src/_pytest/pytester.py]: Thanks.

**Pull request merged**